test/ao/sparsity/test_activation_sparsifier.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Owner(s): ["module: unknown"]
+
+from torch.testing._internal.common_utils import TestCase
+import logging
+import torch
+from torch.ao.sparsity.experimental.activation_sparsifier.activation_sparsifier import ActivationSparsifier
+import torch.nn as nn
+import torch.nn.functional as F
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+
+class Model(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.conv1 = nn.Conv2d(1, 32, kernel_size=3)
+        self.conv2 = nn.Conv2d(32, 32, kernel_size=3)
+        self.max_pool1 = nn.MaxPool2d(kernel_size=2, stride=2)
+
+        self.linear1 = nn.Linear(4608, 128)
+        self.linear2 = nn.Linear(128, 10)
+
+    def forward(self, x):
+        out = self.conv1(x)
+        out = self.conv2(out)
+        out = self.max_pool1(out)
+
+        batch_size = x.shape[0]
+        out = out.reshape(batch_size, -1)
+
+        out = F.relu(self.linear1(out))
+        out = self.linear2(out)
+        return out
+
+
+class _ActivationSparsifierUtils:
```

Comment:
It looks like this class is just a collection of functions to test. Can we remove it or pass the dict of functions directly? Not sure you need a factory here

Reply:
The idea was to organize the configs into a separate class for better readability purposes. Moved it to main Test Class.